vtex-apps/reviews-and-ratings
# Feature/reviews 28 bindings filter

#### What problem is this solving?

Adding the Bindings filter feature for the storefront
CASE1: if the store only has one locale, no binding filter will be shown 
CASE2: For stores that have multiple bindings:
                  1. All previous reviews will have a null value for the **locale** field and will be only shown at the default selection 
                  2. All reviews that not be able to read the **locale** field will be assigned to a null value and only shown at the default selection 
                  3. The default selection is based on the site that the user enters from.
                  4. For stores that have the same locales but different country bindings, it will be treated as CASE1.

#### How to test it?

CASE1: test at [https://aurora--sandboxusdev.myvtex.com/2020-amacsa-ph390/p](url), you might want to run [https://aurora--sandboxusdev.myvtex.com/reviews-and-ratings/verify-schema](url) first if no reviews shown.

CASE2: test at [https://aurora--alleop.myvtex.com/frityurnik-furna-bez-maznina-air-fryer-voltz-v51980j-1700w-12-litra-8-programi-taymer-inoks-cheren-p91953/p?__bindingAddress=www.alleop.bg/](url)
